syntax = "proto3";
import "Resource.proto";

message GameServer {
    string IP = 1;
    int32 port = 2;
    int32 resourceLimit =3;
    Resource resource = 4;
}